z_calibration.py

```diff
@@ -326,7 +335,8 @@ def _set_new_gcode_offset(self, offset):
     def calibrate_z(self):
         self.helper.start_gcode.run_gcode_from_command()
         # probe the nozzle
-        nozzle_zero = self._probe_on_z_endstop(self.helper.nozzle_site)
+        nozzle_zero = self._probe_on_z_endstop(self.helper.nozzle_site,
```

Comment:
It's an interesting funny move with this wiggle. But, I think one wiggle after the probing would be enough.